packages/core/src/line-chart.ts

```diff
@@ -218,6 +218,11 @@ export class LineChart extends BaseAxisChart {
 		super.resizeChart();
 	}
 
+	setXScale () {
+		super.setXScale();
+		this.x.padding(0);
```

Comment:
This is good, but can we add a comment to reference back to baseAxis?

Reply:
And an empty line between the 2 calls, to distinguish

Reply:
What kind of comment do you mean? Just one saying that super refers to BaseAxis?

Reply:
no a comment saying why we're overriding the padding, and that it is set in the `setXScale()` in BaseAxis